src/test/java/bio/terra/service/dataset/flight/create/DatasetCreateFlightTest.java

```diff
@@ -40,16 +42,26 @@ void testDatasetCreateFlightAuthorizeBillingSteps(boolean isTDRBillingProfile) {
     inputParameters.put(JobMapKeys.TDR_BILLING_PROFILE_FALLBACK.getKeyName(), isTDRBillingProfile);
     var flight = new DatasetCreateFlight(inputParameters, context);
 
+    List<String> stepNames = FlightTestUtils.getStepNames(flight);
+
     if (isTDRBillingProfile) {
       assertThat(
           "Dataset creation flight locates the billing info and then has two optional steps",
```

Comment:
This looks like a copy/paste from a different test. Personally I would omit the `reason` in most cases but if it's included it should match the test case.

Reply:
I piggybacked on our existing test case. Should I create a new one?